src/components/ripple/ripple.ts

```diff
@@ -0,0 +1,173 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  HostBinding,
+  Input,
+  OnChanges,
+  OnDestroy,
+  OnInit,
+  SimpleChange,
+  ViewEncapsulation,
+} from '@angular/core';
+import {
+  MdInkRippleManager,
+  ForegroundRipple,
+  ForegroundRippleState,
+} from './ripple-manager';
+
+
+@Component({
+  moduleId: module.id,
+  template: `<div class="md-ripple-background"></div>`,
+  selector: 'md-ink-ripple',
+  styleUrls: ['ripple.css'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MdInkRipple implements OnInit, OnDestroy, OnChanges {
+  /**
+   * The element that triggers the ripple when click events are received. Defaults to the parent
+   * of the <md-ink-rippple>.
+   */
+  @Input('trigger') trigger: Element;
```

Comment:
You don't need the string argument to `@Input` when it's the same thing as the property name, e.g.:
```ts
@Input() trigger: Element;
```